README.md

```diff
@@ -118,6 +124,15 @@ Parameters:
 - `role`: Name of the role, e.g: `caseworker-divorce`.
 - `classification`: Optional. One of `PUBLIC`, `PRIVATE` or `RESTRICTED`. Defaults to `PUBLIC`.
 
+#### Note:
+If you get `{"timestamp":"2019-07-16T09:11:57.403+0000","status":403,"error":"Forbidden","message":"Access Denied","path":"/api/user-role"}` error you may need to install the `jq` library like so:
```

Comment:
There seems to be a confusiopn here. That http 403 is not about jq installation.

Reply:
hmm that is strange, do we know why the 403 happens? I had a 403, I installed the jq library and ran the same command and it worked after that